sky/clouds/service_catalog/oci_catalog.py

```diff
@@ -5,48 +5,86 @@
 
 History:
  - Hysun He (hysun.he@oracle.com) @ Apr, 2023: Initial implementation
+ - Hysun He (hysun.he@oracle.com) @ Jun, 2023: Reduce retry times by
+   excluding those unsubscribed regions.
 """
 
 import typing
 import logging
+import threading
 from typing import Dict, List, Optional, Tuple
+
+from pandas import DataFrame
```

Comment:
Since this is only used for type hinting, we should import this model under the `typing.TYPE_CHECKING` condition in L23.
Also, we should import modules instead of the class, i.e. `import pandas as pd`

Reply:
ok